README.md

```diff
@@ -7,7 +7,7 @@
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
-- [📖 24hours\Doctor\_back\_end ](#-24hours_doctor_back_end-)
+- [📖 24hours\Doctor_back_end ](#-24hours_doctor_back_end-)
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate. 👍